job/server/src/main/java/alluxio/job/plan/stress/StressBenchDefinition.java

```diff
@@ -101,7 +100,7 @@ public StressBenchDefinition() {
       throws Exception {
     Map<String, MountPointInfo> mountTable = runTaskContext.getFileSystem().getMountTable();
     for (Map.Entry<String, MountPointInfo> entry : mountTable.entrySet()) {
-      if (PathUtils.hasPrefix(ufsUri, entry.getKey())) {
+      if (ufsUri.startsWith(entry.getValue().getUfsUri())) {
```

Comment:
Is it possible that I have mount points `hdfs://a` and `hdfs://a/b` and they have different configurations (maybe access control or UGI)?

Reply:
they can't both be mounted, because that would violate our mounting rules.